ls1intum/Artemis
# [Bugfix] Fix graphs on course statistics by taking bonus points into account

### Checklist

### Motivation and Context


Graph was a bit broken when bonus points were applied, see screenshot:
![image](https://user-images.githubusercontent.com/41108487/98028723-c9443d00-1e0e-11eb-9f08-fa62bebb5d07.png)



### Description

- Take bonus points into account for all calculations

### Steps for Testing


1. Log in to Artemis
2. Go to Course Overview and enter a course with exercises (different types) with bonus points that are assessed 
3. Click on Statistics
4. Check if the graphs displays the scores and points correctly. You can see the bonus points when hovering over the graph in `Achieved Scores` and the delta of `Points` and `Reachable points` on the right side


### Test Coverage

Nothing changed

### Screenshots



Fixed graphs
![image](https://user-images.githubusercontent.com/41108487/98035970-90f62c00-1e19-11eb-8221-06dbf1a92b33.png)
